R/new.R

```diff
@@ -7,8 +7,8 @@
 #' @export
 #'
 #' @examples
-#' new_dettl("test import")
-new_dettl <- function(name) {
+#' dettl_new("test import")
+dettl_new <- function(name) {
```

Comment:
If you took a `path` argument here too then the examples would be easier.  Can push this into a new PR if you would rather

Reply:
New ticket for that VIMC-2927